### PR TITLE
Added BLT to GSOC 2021 ideas list.

### PR DESCRIPTION
**Pull request by souravbadami**

### CODE REVIEW

File: pages/initiatives/gsoc/gsoc2021ideas.md [hunk 1]
```diff
@@ -521,3 +521,27 @@ It's a blank page to write.
...
 
 ----
+
+### [OWASP Bug Logging Tool](https://github.com/OWASP/BLT)
+
+##### Brief Explanation:
+
+BLT lets anyone report issues they find on the internet. Found something out of place on Amazon.com ? Let them know. Companies are held accountable and shows their response time and history. Get points for reporting bugs and help keep the internet bug free.
+
+Check OWASP WIKI PAGE https://www.owasp.org/index.php/OWASP_Bug_Logging_Tool for some reference;
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2021ideas.md:531]: This needs updating

File: pages/initiatives/gsoc/gsoc2021ideas.md [hunk 2]
```diff
@@ -521,3 +521,27 @@ It's a blank page to write.
 * [Fred Blaise](mailto:fred.blaise@gmail.com )
 
 ----
+
+### [OWASP Bug Logging Tool](https://github.com/OWASP/BLT)
+
+##### Brief Explanation:
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2021ideas.md:527]:
```suggestion
##### Brief Explanation
```

File: pages/initiatives/gsoc/gsoc2021ideas.md [hunk 3]
```diff
@@ -521,3 +521,27 @@ It's a blank page to write.
...
+
+BLT lets anyone report issues they find on the internet. Found something out of place on Amazon.com ? Let them know. Companies are held accountable and shows their response time and history. Get points for reporting bugs and help keep the internet bug free.
+
+Check OWASP WIKI PAGE https://www.owasp.org/index.php/OWASP_Bug_Logging_Tool for some reference;
+
+##### Getting started
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2021ideas.md:533]:
```suggestion
##### Getting Started
```

File: pages/initiatives/gsoc/gsoc2021ideas.md [hunk 4]
```diff
@@ -521,3 +521,27 @@ It's a blank page to write.
...
+* Get in touch with [Sourav Badami](mailto:sourav.badami@owasp.org) to discuss your idea.
+
+##### Expected Results:
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2021ideas.md:536]:
```suggestion
##### Expected Results
```

File: pages/initiatives/gsoc/gsoc2021ideas.md [hunk 5]
```diff
@@ -521,3 +521,27 @@ It's a blank page to write.
...
+* Flutter app for bug hunting and crowd sourcing hunters with bounty support on top of the existing platform.
+* Code follows existing styleguides and passes all existing quality gates regarding code smells, test coverage etc.
+
+##### Knowledge Prerequisites:
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2021ideas.md:540]:
```suggestion
##### Knowledge Prerequisites
```

File: pages/initiatives/gsoc/gsoc2021ideas.md [hunk 6]
```diff
@@ -521,3 +521,27 @@ It's a blank page to write.
...
+* Javascript, Unit/Integration testing, experience with (or willingness to learn) Flutter, Python, Django some security knowledge would be preferable.
+
+##### Mentors:
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2021ideas.md:543]:
```suggestion
##### Mentors
```

File: pages/initiatives/gsoc/gsoc2021ideas.md (hunk 1 above)
Comment (kingthorin) [reply pages/initiatives/gsoc/gsoc2021ideas.md:531]: Seems you need to actuallynuodate your project page too: https://owasp.org/www-project-bug-logging-tool/

**Review by kingthorin — CHANGES_REQUESTED** [review]

File: pages/initiatives/gsoc/gsoc2021ideas.md (hunk 1 above)
Comment (souravbadami) [reply pages/initiatives/gsoc/gsoc2021ideas.md]: Removed this reference. Github project page is referenced with the title. Will update the OWASP project page soon.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**